package.json

```diff
@@ -72,7 +73,8 @@
     "rules": {
       "guard-for-in": "off",
       "unicorn/prefer-includes": "off"
-    }
+    },
+    "ignores": ["types"]
```

Comment:
this is because `xo` complains about using `Omit` - added `ignores` like [here](https://github.com/syntax-tree/xast-util-sitemap/blob/main/package.json#L83-L85)